RELEASING.md

```diff
@@ -35,5 +35,19 @@ Failure to do so will leave things in a broken state.
     git push upstream
     ```
 
+## Versioning:
+
+Once tag is created, it's time to use that tag for Runtime Versioning
+
+1. Create a new brach for updating version information in `sdk/src/version.cc`
```

Comment:
At what point do we update the version in [api/include/opentelemetry/version.h](https://github.com/open-telemetry/opentelemetry-cpp/blob/master/api/include/opentelemetry/version.h#L6)? Is this automated?

Reply:
version.h has extern declarations, so its not needed to modify it. Only version.cc is recreated.

Reply:
I mean the `version.h` file in the API directory (not the SDK one). See the link above.

Reply:
Ok got it. Didn't check the link initially. Agree that needs to be changed too. I will do required changes.

Reply:
Have updated the script to modify api version.h.